front/src/common/Collapsable.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useEffect, useState, type FC, type HTMLAttributes } from 'react';
+
+import { ChevronDown, ChevronUp } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import { omit } from 'lodash';
+import { useTranslation } from 'react-i18next';
+
+interface CollapsableProps extends Omit<HTMLAttributes<unknown>, 'children' | 'collapsed'> {
+  children: [JSX.Element, JSX.Element];
+  collapsed: boolean;
+}
+const Collapsable: FC<CollapsableProps> = (props) => {
+  const { children, collapsed, className } = props;
```

Comment:
Wouldn't it be possible to initialize it to false so we don't have to always pass it in props ?

```suggestion
  const { children, collapsed = false, className } = props;
```